publisher/publisher.go

```diff
@@ -196,19 +218,21 @@ func (pub *Impl) SubmitToSingleCT(
 		return err
 	}
 
-	stats := pub.stats.NewScope(ctLog.statName)
-	stats.Inc("Submits", 1)
```

Comment:
I think we're losing the total number of submissions stat in this conversion. I only see a prom stat for the # of errors. Did you intentionally omit the # of submissions? It might be useful to have that to compare to the # of errors.

Reply:
Histograms also provide a count of samples, so providing a counter duplicates this.